agents.py

```diff
@@ -125,55 +126,62 @@ def program(percept):
         percepts.append(percept)
         action = table.get(tuple(percepts))
         return action
+
     return program
 
 
-def RandomAgentProgram(actions):
-    "An agent that chooses an action at random, ignoring all percepts."
+def random_agent_program(actions):
```

Comment:
surely they are but in factory methods also standard way is function name should start with lowercase